components/navigation/ui/src/main/resources/PhenoTips/LiveTableResults.xml

```diff
@@ -474,8 +476,46 @@
     ##
     ## Object properties from now on
     ##
-    ## Numbers, exact match
-    #elseif($propType == 'NumberClass' || $propType == 'BooleanClass')
+    ## Numbers, exact match or min/max filters
+    #elseif($propType == 'NumberClass')
+      #livetable_getTableName($colname)
+      #set($fromSql = "${fromSql}, ${tableName} ${safe_tableAlias}")
+      #set($whereSql = "${whereSql} and ${targetedObject}.id=${safe_tableAlias}.id.id and ${safe_tableAlias}.id.name = ?")
+      #set($discard = $whereParams.add("${colname}"))
+      ## Converts each used value into correct type
+      #set($numberType = $propClass.get($colname).getProperty('numberType').getValue())
+      #if($numberType == 'integer')
+        #set($typedFilterValue = $mathtool.toInteger($filterValue))
+        #set($typedFilterValueLowLimit = $mathtool.toInteger($filterValueLowLimit))
+        #set($typedFilterValueHighLimit = $mathtool.toInteger($filterValueHighLimit))
+      #elseif($numberType == 'float')
+        #set($typedFilterValue = $mathtool.toDouble($filterValue).floatValue())
+        #set($typedFilterValueLowLimit = $mathtool.toDouble($filterValueLowLimit).floatValue())
+        #set($typedFilterValueHighLimit = $mathtool.toDouble($filterValueHighLimit).floatValue())
+      #elseif($numberType == 'double')
+        #set($typedFilterValue = $mathtool.toDouble($filterValue))
+        #set($typedFilterValueLowLimit = $mathtool.toDouble($filterValueLowLimit))
+        #set($typedFilterValueHighLimit = $mathtool.toDouble($filterValueHighLimit))
+      #else ## Long
+        #set($typedFilterValue = $mathtool.toDouble($filterValue).longValue())
+        #set($typedFilterValueLowLimit = $mathtool.toDouble($filterValueLowLimit).longValue().valueOf($filterValueLowLimit))
+        #set($typedFilterValueHighLimit = $mathtool.toDouble($filterValueHighLimit).longValue().valueOf($filterValueHighLimit))
+      #end
+      ## Checks present filter value
+      #if ($filterValue != '')
+        #set($whereSql = "${whereSql} and ${safe_tableAlias}.value = ?")
+        #set($discard = $whereParams.add($typedFilterValue))
+      #else ## filters by min and max values
+        #if ($filterValueLowLimit != '')
```

Comment:
Shouldn't this instead be `$typedFilterValueLowLimit` to cover the case when the input is not a valid number?